core/bounds.gms

```diff
@@ -505,28 +505,6 @@ vm_emiFgas.fx(ttot,all_regi,all_enty) = f_emiFgas(ttot,all_regi,"%c_SSP_forcing_
 display vm_emiFgas.L;
 
 
-*** Bugfix. For some reason the model cannot reduce the production of district heating to 0
```

Comment:
Can this really be removed, or does it have to be refactored?

Reply:
@robertpietzcker Can you comment on this? Can all of this deleted, or should we keep the fix for "feheb" only, or does this have to be refactored?

Reply:
if the runs work without I think it can be deleted. as the description said, it was never clear why this was needed - the model should have been able to stay at 0 anyway